packages/admin-portal/src/components/ui/approval-modal/approve-revision-modal.tsx

```diff
@@ -42,13 +43,11 @@ export const ApproveRevisionModal: React.FunctionComponent<ApproveRevisionModalP
   const revisionDetail = useSelector(selectAppRevisionDetail)
   const session = useReapitConnect(reapitConnectBrowserSession)
   const { email, name } = session?.connectSession?.loginIdentity || {}
+  const formState = useSelector(selectAppRevisionFormState)
+  const { appId, id: appRevisionId } = (revisionDetail?.data?.data || {}) as AppRevisionModel
 
-  const { approveFormState } = revisionDetail
-
-  const { appId, id: appRevisionId } = revisionDetail?.revisionDetailData?.data || {}
-
-  const isLoading = approveFormState === 'SUBMITTING'
-  const isSuccessed = approveFormState === 'SUCCESS'
+  const isLoading = formState === 'SUBMITTING'
```

Comment:
could you use isLoading from reducer instead of check formState

Reply:
This is request state, and required 'SUBMITTING', 'SUCCESS'. Not just boolean. Not sure you mean